04.class/operateInterface.js

```diff
@@ -0,0 +1,97 @@
+import readline from "readline";
+import enquirer from "enquirer";
+
+const { Select } = enquirer;
+export default class OperateInterface {
+  constructor(memoOperation) {
+    this.memoOperation = memoOperation;
+  }
+
+  async listMemos() {
+    try {
+      const rows = await this.memoOperation.fetchMemos();
+      rows.forEach((row) => {
+        console.log(row.content.split("\n")[0]);
+      });
+    } catch (err) {
+      console.error(err);
+    }
+  }
+
+  async readMemos() {
+    try {
+      const result = await this.#selectMemosFromList("see");
+      console.log(result);
+    } catch (err) {
+      console.error(err);
+    }
+  }
+
+  async deleteMemo() {
+    try {
+      const result = await this.#selectMemosFromList("delete");
+      this.memoOperation.removeMemo(result);
+    } catch (err) {
+      console.error(err);
+    }
+  }
+
+  async writeMemo() {
+    try {
+      const content = await this.#createMemoPrompt();
+      await this.memoOperation.addMemo(content);
+    } catch (err) {
+      console.error(err);
+    }
+  }
+
+  async #selectMemosFromList(action) {
```

Comment:
```suggestion
  async #selectMemoFromList(action) {
```

取ってくるメモが1件なら単数形の方が自然かなー

Reply:
ずっと複数のメモを取って来ると思っていましたが、selectするのは一件のメモということですね💦
単数形に修正します！